src/ui/wxWidgets/PasswordSafeSearch.cpp

```diff
@@ -620,7 +623,9 @@ void PasswordSafeSearch::OnChar(wxKeyEvent& event)
     HideSearchToolbar();
   }
   else if ((event.GetModifiers() == wxMOD_CONTROL) && 
-    ((event.GetKeyCode() == wxT('c')) || (event.GetKeyCode() == wxT('C')))) {
+    ((event.GetKeyCode() == wxT('c')) || (event.GetKeyCode() == wxT('C')) || 
+     (event.GetKeyCode() == wxT('u')) || (event.GetKeyCode() == wxT('U')) ||
+     (event.GetKeyCode() == wxT('x')) || (event.GetKeyCode() == wxT('X')))) {
```

Comment:
Nit: this list of keys might grow in the future. Can we do something like `wxT('CcUuXx').contains(event.GetKeyCode())` (sorry, don't know the right string operator to use).